app/renderer/abr-document.js

```diff
@@ -298,19 +297,32 @@ AbrDocument.prototype = {
         var filenameCapture = /([\w][\w\s-]*)/; // Captures are expensive, so use it only if required
         var specialChar = /[^\w\s-]/g;
 
-        if (this.cm.doc.lineCount() < 1) {
+        if (this.getData().trim().length < 1) {
```

Comment:
I think this is useless since in this case the function won't return any titleLine at the end anyway.